lib/jobs/isc-dhcp-lease-poller.js

```diff
@@ -144,6 +144,10 @@ function iscDhcpLeasePollerFactory(
         */
         var split = data.split('\n');
         var lease;
+        var todaysDate = new Date();
```

Comment:
Can we call this currentTime or something? `todaysDate` just makes me think of a date at the day level, not one that's precise to the millisecond.